README.md

```diff
@@ -36,10 +41,10 @@ We have released the GDK for Unreal this early in development because we want yo
 ## Where to get the GDK and starter projects
 The GDK and its starter projects are available on GitHub.
 * [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
+* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
 * [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
-* [Third Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
-* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
-* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) </br>
+* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
+* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)</br>
 **NOTE:** This link may give you a 404. 
```

Comment:
This note needs to sit under the UE4 link - it's why it was last before - I think Giray re-ordered but the 404 note doesn't make sense now, I have just realised.

Reply:
Yes